test/extended/testdata/builds/custom-build/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM docker.io/centos:7
+RUN yum install -y buildah
```

Comment:
I'm inclined to replace these two lines with `FROM quay.io/buildah/stable:latest` so we run our "custom build with buildah" e2e using the latest stable buildah image.

@TomSweeneyRedHat @rhatdan @nalind thoughts?

Reply:
If yes, @bmcelvee we would also want to update our docs, since QE pulled this straight from there.

Reply:
@adambkaplan  Updated it according comments, yes I can found image quay.io/buildah/stable:latest, @TomSweeneyRedHat @rhatdan @nalind if we need use the image: quay.io/buildah/stable:latest? if so we will update 4.1 docs. hope your response, thanks.

Reply:
Just back from enjoying PTO.  The quay.io/buildah/stable:latest makes sense to me.  The only caveat is it's built on top of Fedora and not CentOS.  New versions of the stable builds for Buildah land in Fedora first so there might be a few days of version skew between CentOS and Fedora, IDK if that matters or not.

Also, it looks like  `test/extended/testdata/builds/custom-build/Dockerfile.sample` is still using CentOS, not sure if you want to convert that too or not?

Reply:
Hopefully we can move to UBI8 once RHEL8.1 is release.

Reply:
@TomSweeneyRedHat  IMO, no need to convert CentOS to quay.io/buildah/stable:latest, the Dockerfile.sample just create space for custom build used, @adambkaplan  how about do you opinion?

Reply:
@wewang58 I agree with you - keep `Dockerfile.sample` as is.